### PR TITLE
Eliminate pl variable score

### PR DESCRIPTION
**Pull request by tbretl**
Elements are now responsible for displaying their own partial scores.

### CODE REVIEW

File: elements/pl_matrix_input/pl_matrix_input.mustache [hunk 1]
```diff
@@ -14,11 +14,14 @@
 <div class="input-group">
     {{#label}}<label class="input-group-addon" id="basic-addon2">{{label}}</label>{{/label}}
     <input name="{{name}}" type="text" class="form-control" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
-    <span class="input-group-btn" id="basic-addon1">
+    <div class="input-group-btn" id="basic-addon1">
         <a class="btn btn-default" type="button" data-toggle="popover" data-html="true" title="MATLAB Matrix" data-content="{{info}}" data-placement="auto left" data-trigger="focus" tabindex="0">
             <i class="fa fa-question-circle" aria-hidden="true"></i>
+            {{#correct}}&nbsp<span class="label label-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
```
Comment (mwest1066) [inline elements/pl_matrix_input/pl_matrix_input.mustache]: I think HTML entities need a semi-colon? Like `&nbsp;`? See https://developer.mozilla.org/en-US/docs/Glossary/Entity

File: elements/pl_checkbox/pl_checkbox.py [hunk 2]
```diff
@@ -102,11 +106,29 @@ def render(element_html, element_index, data):
...
+            try:
+                score = float(score)
+                if score >= 1:
+                    html = html + '&nbsp<span class="label label-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>'
+                elif score > 0:
+                    html = html + '&nbsp<span class="label label-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {:d}%</span>'.format(math.floor(score * 100))
+                else:
+                    html = html + '&nbsp<span class="label label-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>'
+            except:
+                raise ValueError('invalid score' + score)
```
Comment (mwest1066) [inline elements/pl_checkbox/pl_checkbox.py]: The `submission` panel render doesn't include the score.

File: elements/pl_matrix_input/pl_matrix_input.mustache (hunk 1 above)
Comment (tbretl) [reply elements/pl_matrix_input/pl_matrix_input.mustache]: Sorry, fixed.

File: elements/pl_checkbox/pl_checkbox.py (hunk 2 above)
Comment (tbretl) [reply elements/pl_checkbox/pl_checkbox.py]: I don't know how I missed this. Fixed.

**Pull request merged**